plugins/Example/main.lua

```diff
@@ -15,6 +16,16 @@ end
 
 -- Callback if the menu item is executed
 function exampleCallback()
+  app.openDialog("Test123", {[1] = "Yes", [2] = "No"}, "dialogCallback");
```

Comment:
I would actually be better to use it that way (to avoid having the order of the buttons arbitrary)
```suggestion
  app.openDialog("Test123", {"Yes", "No"}, "dialogCallback");
```